rules-tests/Php80/Rector/Identical/StrEndsWithRector/Fixture/fixture.php.inc

```diff
@@ -25,6 +29,10 @@ class Fixture
         $isMatch = str_ends_with($haystack, $needle);
 
         $isMatch = str_ends_with($haystack, $needle);
+
+        $isMatch = str_ends_with($haystack, $needle);
+
+        $isMatch = str_ends_with($haystack, $needle);
```

Comment:
Please always add a new fixture, instead of modifying existing one. That way we can bind the feature directly to test file in case of regression :+1:

Reply:
I have extracted new fixtures, hope this matches what you were thinking of. CI is failing now but it seems to be because of some composer.json package mismatch